diesel_derives/src/queryable_by_name.rs

```diff
@@ -78,8 +78,8 @@ fn sql_type(field: &Field, model: &Model) -> syn::Type {
             parse_quote!(diesel::dsl::SqlTypeOf<#table_name::#column_name>)
         } else {
             let field_name = match field.name {
-                FieldName::Named(ref x) => x.as_ref(),
```

Comment:
This could also be `to_string` instead of creating another `Ident`. That would make it clearer that it's not going to be used in a `quote!` call.